Description: Add missing lib to link, resolving FTBFS on arm64.
Author: Daniel T Chen <crimsun@ubuntu.com>

Index: ike-2.2.1+dfsg/source/libith/CMakeLists.txt
===================================================================
--- ike-2.2.1+dfsg.orig/source/libith/CMakeLists.txt	2013-10-18 22:12:09.000000000 -0400
+++ ike-2.2.1+dfsg/source/libith/CMakeLists.txt	2014-02-17 11:53:07.064741056 -0500
@@ -20,6 +20,7 @@
 
 	target_link_libraries(
 		ss_ith
+		pthread
 		rt )
 
 endif( PATH_LIB_RT )
